templates/base.html

```diff
@@ -102,6 +107,33 @@
         </script>
         {% endblock script %}
         {% include '_includes/stat_counter.html' %}
+
+        {% if PHOTOS_LIGHTBOX and article and article.photo_gallery %}
+        <!-- 'photo' Gallery support -->
+        <!-- Magnific Popup core JS file -->
+        <script src="{{ SITEURL }}/{{ THEME_STATIC_DIR }}/magnific/magnific-popup.js"></script>
+        <script>
+            $('.gallery').magnificPopup({
+                delegate: 'a',
+                type: 'image',
```

Comment:
Move this script to article.html `script block`

See https://github.com/Pelican-Elegant/elegant/blob/master/templates/article.html#L119-L123

Also I suggest putting this code in other file and then include it. This will keep the code clean.